editor/src/uuiui/radix-components.tsx

```diff
@@ -103,7 +104,7 @@ export const DropdownMenu = React.memo<DropdownMenuProps>((props) => {
   }, [])
   const onEscapeKeyDown = React.useCallback((e: KeyboardEvent) => e.stopPropagation(), [])
 
-  const [open, setOpen] = React.useState(false)
+  const [open, setOpen] = useIsDropdownMenuOpen(props.isOpen ?? null)
```

Comment:
Moved into a dedicated hook so we can keep track of the `useEffect` (which I don't think can be avoided).